### ISSUE TITLE
Need a payload strict matcher feature for zerocode assertions

### ISSUE THREAD

**Issue by officiallysameer** [authorjapps/zerocode#312]
I need a `strict` matching capability for API contracts in my project.

e.g.
Actual payload:
```json
{
	"id": 123,
	"sanctionList": "GLOBAL"
}
```

The below expected payload will `pass`
```
{
	"id": 123,
	"sanctionList": "GLOBAL"
}
```

The below expected payload(s) 
should `fail`.
```
{
	"id": 123
}
```
should `fail`.
```
{
	"sanctionList": "GLOBAL"
}
```
should `fail`.
```
{
	"id": 123,
	"sanctionList": "GLOBAL",
	"bank": "HSBC"
}
```

**Comment by authorjapps** [comment]
We can go for STRICT mode comparison something simillar to `SkyScreamer`.
> JSONCompareMode#STRICT

**Comment by nirmalchandra** [comment]
@officiallysameer , can you please bring up a Wiki page on this?

**Comment by authorjapps** [comment]
Available in `1.3.14` and higher

**Comment by officiallysameer** [comment]
Wiki link
https://github.com/authorjapps/zerocode/wiki/Strict-Mode-Payload-Comparison
